cmd/algorand-indexer/reset.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/algorand/indexer/config"
+	"github.com/algorand/indexer/idb"
+	"github.com/algorand/indexer/importer"
+)
+
+var (
+	andRebuild bool
+)
+
+var resetCmd = &cobra.Command{
+	Use:   "reset",
+	Short: "run indexer reset",
+	Long:  "run indexer reset. Serve api on HTTP.",
```

Comment:
could you maybe add that this is a developer's tool only?

Reply:
This was partly copy-paste-o from the daemon command. Long text is now explanatory and cautionary.